addons/l10n_cz/data/template/account.tax-cz.csv

```diff
@@ -1,39 +1,67 @@
-"","","","","","","","","","tax","invoice","chart_cz_343215",""
-"","","","","","","","","","base","refund","",""
-"","","","","","","","","","tax","refund","chart_cz_343215",""
-"vyc_tuz_12","DPH na výstupu 12%","","12%","12.0","percent","sale","0","tax_group_vat_12","base","invoice","",""
-"","","","","","","","","","tax","invoice","chart_cz_343212",""
-"","","","","","","","","","base","refund","",""
-"","","","","","","","","","tax","refund","chart_cz_343212",""
-"vyc_tuz_0","DPH na výstupu 0%","","0%","0.0","percent","sale","","tax_group_vat_0","base","invoice","",""
-"","","","","","","","","","tax","invoice","",""
-"","","","","","","","","","base","refund","",""
-"","","","","","","","","","tax","refund","",""
-"vsc_tuz_21","DPH na vstupu 21%","","21%","21.0","percent","purchase","0","tax_group_vat_21","base","invoice","",""
-"","","","","","","","","","tax","invoice","chart_cz_343121",""
-"","","","","","","","","","base","refund","",""
-"","","","","","","","","","tax","refund","chart_cz_343121",""
-"vsc_tuz_15","DPH na vstupu 15%","","15%","15.0","percent","purchase","0","tax_group_vat_15","base","invoice","",""
-"","","","","","","","","","tax","invoice","chart_cz_343115",""
-"","","","","","","","","","base","refund","",""
-"","","","","","","","","","tax","refund","chart_cz_343115",""
-"vsc_tuz_12","DPH na vstupu 12%","","12%","12.0","percent","purchase","0","tax_group_vat_12","base","invoice","",""
-"","","","","","","","","","tax","invoice","chart_cz_343112",""
-"","","","","","","","","","base","refund","",""
-"","","","","","","","","","tax","refund","chart_cz_343112",""
-"vyc_dod_eu","Dodání do EU","","0%","0.0","percent","sale","","tax_group_vat_0","base","invoice","",""
-"","","","","","","","","","tax","invoice","",""
-"","","","","","","","","","base","refund","",""
-"","","","","","","","","","tax","refund","",""
-"vsc_nad_eu","Pořízení z EU","","21%","21.0","percent","purchase","","tax_group_vat_21","base","invoice","",""
-"","","","","","","","","","tax","invoice","chart_cz_343121",""
-"","","","","","","","","","tax","invoice","chart_cz_343221","-100"
-"","","","","","","","","","base","refund","",""
-"","","","","","","","","","tax","refund","chart_cz_343121",""
-"","","","","","","","","","tax","refund","chart_cz_343221","-100"
+"id","name","invoice_label","tax_scope","type_tax_use","amount","invoice_repartition_line_ids/factor_percent","invoice_repartition_line_ids/repartition_type","invoice_repartition_line_ids/document_type","invoice_repartition_line_ids/factor","invoice_repartition_line_ids/account_id","invoice_repartition_line_ids/tag_ids","refund_repartition_line_ids/factor_percent","refund_repartition_line_ids/repartition_type","refund_repartition_line_ids/document_type","refund_repartition_line_ids/factor","refund_repartition_line_ids/account_id","refund_repartition_line_ids/tag_ids","price_include","amount_type","tax_group_id","name@cs","invoice_label@cs"
```

Comment:
it's missing the description column ? even if the column would be empty i would not remove it, it can be useful